ui/app/build-configs/directives/pnc-build-script-forbidden-validator.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2014-2020 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+(function () {
+
+  var module = angular.module('pnc.build-configs');
+
+  /**
+   * Check whether Build Script input is not containing forbidden commands.
+   */
+  module.directive('pncBuildScriptForbiddenValidator', function () {
+
+      var buildScriptForbiddenChecker = function(value) {
+        var FORBIDDEN_ARGS = [`-X`, `"-X"`, `'-X'`, `--debug`, `"--debug"`, `'--debug'`];
```

Comment:
1) It seems that cases like `-Xabc` would be forbidden too, but they should be allowed, see https://github.com/project-ncl/pnc-web-ui-react/pull/446/files

There is following solution:
```js
[` -X `, ` "-X" `, ` '-X' `, ` --debug `, ` "--debug" `, ` '--debug' `];
```

and then:

```js
const lineWithEndSpace = `${line} `;
```


2) `value` seems to be whole buildScript, splitting it should be considered:

```js
const lines = buildScript.toLowerCase().split('\n');
```

and then perform checker itself it for each line.

At some point we should adjust also `pnc-build-script-validator`